bin/build-hdfs

```diff
@@ -104,7 +114,7 @@ cd -
 echo "Compressing to $EXECUTOR.tgz"
 rm -f $BUILD_CACHE_DIR/$EXECUTOR.tgz
 cd $BUILD_CACHE_DIR
-tar czf $EXECUTOR.tgz $EXECUTOR
```

Comment:
This will implicitly change the structure of the executor binary. Is that intentional?